Rebalances security loadouts and equipment

TsunamiAnt opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

This came up in a discussion I had with ruko and bacon, and it sounded fun. So here goes:

- Adjusted security belt slots to 5
- Removed most starting gear except for cuffs, taser, and baton from the belt
- Since there is now space, moved the magazine and taser cartridge they spawn with into the belt, filling it up fully.
- The webbing has been altered to not allow large items inside such as batons.
- Tasers were made smaller(why were they as big as a baton?) so they can fit into webbings.
- Webbings have had their maximum slot count increased.
- Some general loadout changes as well as minor tweaks to item sizes occurred due to these alterations.

Now the webbing has more slots and can be used to hold a lot of small stuff, as opposed to the belt which has less slots than it but can hold bigger things.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

These changes aim to provide a more balanced and logical distribution of equipment for security personnel, making their roles more effective and enjoyable.

It also fixes inconsistencies, like the taser and it's ammo being weirdly huge.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

<img width="966" height="751" alt="image" src="https://github.com/user-attachments/assets/30f48128-beab-4b1c-ab25-4d390f049eb5" />

<img width="1545" height="1246" alt="image" src="https://github.com/user-attachments/assets/735e4fe1-7cfc-4025-a958-a565d15ac667" />

</details>

## Changelog
:cl:
balance: Rebalanced security loadouts. Security belts now have adjusted item spawns and storage capacities.
balance: Moved the backpack magazine and taser cartridge into their belts whenever feasable.
tweak: Modified the weight class of taser cartridges and taser itself.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

TsunamiAnt committed:
Rebalances security loadouts and equipment

Reconfigures security belts to have more sensible starting equipment.

Adjusts starting equipment for security roles by removing items from backpacks and directly equipping them on the belt.

Tweaks taser stats to have a more appropriate weight class.

PowerfulBacon reviewed (changes requested):
Brings taser size up to large, in line with all other guns, bring taser cartridges up to a small item. If you want webbings to hold them then let them hold all large items. If not, let it be an item for carrying more of your less important gear. I would side with allowing large items in them so that they are a direct upgrade over the belt but I don't care which direction it goes.

While I don't agree with the necessity for benefit of making any of these sec changes, I have always disliked the larger size of the belt since the new changes and how disorganised sec inventories end up looking compared to before so I would be happy to bring back the old belt/webbing behaviours.

TsunamiAnt committed:
Balances security belt and taser stats.

Adjusts security belt storage and taser weight classes for balance.

Removes a redundant max specific storage definition for security belts.

Increases the taser's projectile weight class to small.

Increases the taser's weapon weight class to large.

PowerfulBacon commented on code/game/objects/items/storage/belt.dm:
```diff
@@ -428,6 +428,11 @@
+/obj/item/storage/belt/security/webbing/Initialize(mapload)
+	. = ..()
+
+	atom_storage.max_slots = 9
```
Shouldn't this be 7 so it's only 1 row instead of leaking over?

TsunamiAnt replied:
7 is not really a lot. But i can change it if you wanna

Rukofamicom replied:
This was one of the things I said in discord, the webbing needs to be either 7 or 14 for a clean UI.

7 is good if it holds large items
14 if you want it to only hold pocket sized ones

This PR can be considered as having my approval when this is changed

PowerfulBacon replied:
I think 7 is good now that it can hold large items again.

TsunamiAnt replied:
o7 aye aye

TsunamiAnt committed:
Decrease max slots of security webbing from 9 to 7

TsunamiAnt committed:
Merge branch 'master' into securityloadout
